### DISCUSSION TITLE
Keyboard shortcut to autofill the code for the current website?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [Authenticator-Extension/Authenticator#829] [Ideas]
Basically the title, this would be super useful I think! I use bitwarden a lot in combination with authenticator and this would be really powerful in combination with Bitwarden's autofill. This should be a keyboard shortcut set in `chrome://extensions/shortcuts` I think. 

Thanks for your time!

**Comment by lsloan** [comment]
Authenticator has an Autofill feature, but it may not work as you expect it.  In my experience, I found this to work well…

1. In Google Chrome, set "Keyboard shortcuts" (chrome://extensions/shortcuts) for the Authenticator extension to "Activate the extension" with some key combination you like.  I used <kbd>Option</kbd><kbd>Shift</kbd><kbd>A</kbd>.
2. In Authenticator's "Settings" under "Preferences" enable the option for "Use Autofill".  This doesn't seem to make Authenticator automatically fill in codes when a form is detected on the page.  It's not like Chrome's password autofill, but it helps with the next steps.
3. When you are prompted by some website for an authentication code and the input field has keyboard focus, press the keyboard shortcut set in step 1.  When the Authenticator window appears, use the cursor up and down keys on the keyboard to highlight the needed code.  When the code is highlighted, press the <kbd>Return</kbd> key to copy it, which will also make Authenticator fill the code into input field.

**Reply by Explosion-Scratch** [reply]
Yeah, that's the current workaround. I ended up using a self hosted docker versoin of [Vaultwarden](https://github.com/dani-garcia/vaultwarden) (I backup the data to a GitHub repo via a cron job) and now Bitwarden copies the totp code when I use the autofill feature. Self hosting vaultwarden gives you all the premium features of Bitwarden too such as files, auth codes, etc

**Reply by rkristelijn** [reply]
Awesome! I couldn't set `Option` `Shift` `A`, only `Option` `A`. AFAIK it is not used for anything in Google Chrome, tested it and it worked nicely!

<img width="1196" height="650" alt="image" src="https://github.com/user-attachments/assets/1f5f5b01-0d41-446c-abc3-06c1219b2c31" />
